app/src/main/java/com/example/namp5/services_notification/MyService.java

```diff
@@ -0,0 +1,235 @@
+package com.example.namp5.services_notification;
+
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Intent;
+import android.media.MediaPlayer;
+import android.os.Binder;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.TaskStackBuilder;
+import android.widget.RemoteViews;
+
+/**
+ * Created by namp5 on 11/30/2018.
+ */
+
+public class MyService extends Service implements MusicInterface, MediaPlayer.OnCompletionListener {
+    public static final String EXTRA_REQUEST_CODE = "REQUEST CODE";
+    public static final int VALUE_NEXT_SONG = 1112;
+    public static final int VALUE_PREVIOUS_SONG = 1113;
+    public static final int VALUE_PLAY_SONG = 1114;
+    public static final int NUMBER_ONE = 1;
```

Comment:
```suggestion
    public static final int CHANGE_POSITION = 1;
```
hoặc đặt tên có ý nghĩ hơn